### PR TITLE
remove excluded deposits

### PR DESCRIPTION
**Pull request by Joeysantoro**
Excluded deposits provide too many edge cases and too much power to the guardian. This PR removes this ability. The next PR will provide an alternative which is simpler and less risky

### CODE REVIEW

File: test/unit/oracle/CollateralizationOracle.test.ts [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-const CollateralizationOracle = artifacts.readArtifactSync('CollateralizationOracle');
-const MockPCVDepositV2 = artifacts.readArtifactSync('MockPCVDepositV2');
-const MockOracleCoreRef = artifacts.readArtifactSync('MockOracleCoreRef');
-const MockERC20 = artifacts.readArtifactSync('MockERC20');
-const IFei = artifacts.readArtifactSync('IFei');
-
 const e18 = '000000000000000000';
 
-describe('CollateralizationOracle', function () {
+describe.only('CollateralizationOracle', function () {
```
Comment (ditchfieldcaleb) [inline test/unit/oracle/CollateralizationOracle.test.ts:8]: Don't forget to remove this

**Review by ditchfieldcaleb — COMMENTED** [review]
See single comment

File: test/unit/oracle/CollateralizationOracle.test.ts (hunk 1 above)
Comment (Joeysantoro) [reply test/unit/oracle/CollateralizationOracle.test.ts:8]: Removed in subsequent PR
